### ISSUE TITLE
Is this repo maintained?

### ISSUE THREAD

**Issue by enkelmedia** [anghelvalentin/CountryValidator#17]
Is this still actively maintained or should it be considered archived?

**Comment by kinguru** [comment]
Seems not. 
Did you find another code which is still supported? 

Thanks

**Comment by enkelmedia** [comment]
@kinguru Nope, I did not - I have a kind of similar need but not exactly so we decided to "roll our own" thing. Hope you find something!

**Comment by tonyqus** [comment]
I checked the nuget release. The last release happened in Nov. 2021

@anghelvalentin Do you have plan to release a new version? There are a few updates after Nov. 2021
